app/jobs/check_published_initiatives.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CheckPublishedInitiatives < ApplicationJob
+  def perform
+    system "rake decidim_initiatives:check_published"
```

Comment:
Prefer to call directly objects as the [rake task](https://github.com/decidim/decidim/blob/1ff32dda20efae8fb2b29ecc2f44c93dba814f1d/decidim-initiatives/lib/tasks/decidim_initiatives.rake#L6) rather than invoking a system command